test/test_scanner.py

```diff
@@ -251,6 +251,7 @@ def test_version_in_package(self, url, package_name, product, version):
                 list_versions.add(product_info.version)
 
         # Make sure the product and version are in the results
+        print(list_products, list_versions)
```

Comment:
Looks like this is a leftover debug statement; you should remove this change from your final PR.